platform/viewer/src/App.js

```diff
@@ -145,7 +138,7 @@ class App extends Component {
     _initServices([
       UINotificationService,
       UIModalService,
-      UIDialogService,
+      [UIDialogService, { testing: true }],
```

Comment:
Should this be committed?

Reply:
That was just a test to see if you were paying attention. Heres your 🍪 ...

No, this was pushed by mistake hehe, removed. thanks!